tests/suites/test_suite_psa_crypto.data

```diff
@@ -6981,27 +6981,27 @@ depends_on:PSA_WANT_ALG_CTR:PSA_WANT_KEY_TYPE_AES
 generate_key:PSA_KEY_TYPE_AES:64:PSA_KEY_USAGE_EXPORT | PSA_KEY_USAGE_ENCRYPT | PSA_KEY_USAGE_DECRYPT:PSA_ALG_CTR:PSA_ERROR_INVALID_ARGUMENT:0
 
 PSA generate key: RSA, 512 bits, good, sign (PKCS#1 v1.5)
-depends_on:PSA_WANT_ALG_RSA_PKCS1V15_SIGN:MBEDTLS_PSA_WANT_KEY_TYPE_RSA_KEY_PAIR_LEGACY:MBEDTLS_GENPRIME
+depends_on:PSA_WANT_ALG_RSA_PKCS1V15_SIGN:MBEDTLS_PSA_WANT_KEY_TYPE_RSA_KEY_PAIR_LEGACY:MBEDTLS_GENPRIME:MBEDTLS_RSA_MIN_KEY_SIZE <= 512
```

Comment:
The correct dependency would be `>= 512`, not `<= 512`. Same with the other test cases below.

Reply:
I want the MBEDTLS_RSA_MIN_KEY_SIZE to be maximum 512 bits, if it's  >= 512 this means that we expect generating a 512 bits key with MBEDTLS_RSA_MIN_KEY_SIZE set to 1024 to be a success which it shouldn't because I shouldn't be able to generate a 512 bits key with a 1024 bit minimum key size

Reply:
Oh, right, duh, this one is a positive test so needs `min <=`. Sorry for the noise.